app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -667,3 +671,70 @@ fun CharSequence.standardize(): String = toString().removeAccents().trim().lower
 inline val AndroidViewModel.context: Context get() = getApplication()
 
 val TextInputEditText.trimmedText inline get() = text?.trim().toString()
+
+fun Context.postfixWithTag(
+    original: CharSequence,
+    @StringRes tagRes: Int,
+    @ColorRes backgroundColorRes: Int,
+    @ColorRes textColorRes: Int,
+    onClicked: (() -> Unit)? = null,
+) = postfixWithTag(original, getString(tagRes), backgroundColorRes, textColorRes, onClicked)
```

Comment:
I'm a bit "lost" by the logic to call getString here, but not getting the color res
I'd either get all the resolved resource here, or passing the res to the function that use it (i.e call getString() in postfixWithTag, then getColor in setTagSpan as it is now)

Reply:
The idea behind calling getString earlier in a kind of "helper" is just to make the whole thing easier to use instead of always having to resolve the string before calling the main helper which takes a String.

Also, using color res all the way down is because contrary to the tag, we never want to give a color which is not in a resource imo. Also there are other parameters such as text size and text font which are resolved at the same time as background and text colors because they're kind of the same thing, they apply to the same thing which is why I think it's ok to resolve them next to each other in the code as well.

Reply:
I still find it more logical to resolves in the function that use it, but as tag is used twice here, I can understand not adding a line to create the val